deploy/kubernetes/filebeat-kubernetes.yaml

```diff
@@ -112,9 +112,16 @@ metadata:
 data:
   filebeat.yml: |-
     filebeat.inputs:
-    - type: container
+    - type: filestream
       paths:
         - /var/log/containers/*.log
+      parsers:
+        - container: ~
+      prospector:
+        scanner:
+          fingerprint.enabled: true
+          symlinks: true
+      file_identity.fingerprint: ~
```

Comment:
Looks good to me, I have not tried this configuration myself but I assume you've tried it, @MichaelKatsoulis .

Reply:
Yes, I have !